<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Saab</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(14,18,31);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Tlwg Typo</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane
{
	border: 1px;
	background: rgb(18, 22, 36);
}

QTabBar::tab
{
	background: rgb(27, 33, 53);
}

QTabBar::tab:selected
{
	background: rgb(27, 33, 70);
}
</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_home">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="12" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_const_4">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>Total vertexes:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_total_poligons">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
color: rgb(142, 143, 148);</string>
          </property>
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_const_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLabel" name="label_const_1">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 30px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>Data obtained from the file:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QLabel" name="label_const_viewer">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 104px;
color: rgb(0, 150, 255);
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt;3DViewer&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;by tonitaga jerlenem rashidag&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_location_model">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
color: rgb(142, 143, 148);</string>
          </property>
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="label_total_vertexes">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
color: rgb(142, 143, 148);</string>
          </property>
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_const_5">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>Total poligons:</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_const_6">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>Location of model:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_name_of_model">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
color: rgb(142, 143, 148);</string>
          </property>
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_const_2">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>Name of model:</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0" colspan="3">
         <widget class="QPushButton" name="pushButton_loadmodel">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 95, 117);
color: rgb(0, 150, 255);
font-size: 40px;</string>
          </property>
          <property name="text">
           <string>Load the Model</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="14" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3dviewer">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>3DViewer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="3">
         <widget class="QLineEdit" name="lineEdit_rotateOZ">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 95, 117);
font-size: 16px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>0.25</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_rotateOZ">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);</string>
          </property>
          <property name="text">
           <string>Rotate Model OZ</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_rotateOY">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);</string>
          </property>
          <property name="text">
           <string>Rotate Model OY</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="lineEdit_rotateOY">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 95, 117);
font-size: 16px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>0.25</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="lineEdit_rotateOX">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 95, 117);
font-size: 16px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>0.25</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="7">
         <widget class="ViewModel" name="openGLWidget"/>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_moveOY">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);</string>
          </property>
          <property name="text">
           <string>Move Model OY</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_moveOY">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 95, 117);
font-size: 16px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>0.25</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QLineEdit" name="lineEdit_scale">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 95, 117);
font-size: 16px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>0.95</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_moveOZ">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);</string>
          </property>
          <property name="text">
           <string>Move Model OZ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_moveOX">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);</string>
          </property>
          <property name="text">
           <string>Move Model OX</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_rotateOX">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);</string>
          </property>
          <property name="text">
           <string>Rotate Model OX</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_moveOX">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 95, 117);
font-size: 16px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>0.25</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_moveOZ">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(83, 95, 117);
font-size: 16px;
color: rgb(0, 150, 255);</string>
          </property>
          <property name="text">
           <string>0.25</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pushButton_scale">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);</string>
          </property>
          <property name="text">
           <string>Scale Model</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 8px;
</string>
          </property>
          <property name="text">
           <string>Only (0, inf)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_saveImage">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);</string>
          </property>
          <property name="text">
           <string>Save image</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_recordScreen">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);</string>
          </property>
          <property name="text">
           <string>Record screen</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_toStartPoint">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);</string>
          </property>
          <property name="text">
           <string>To start point</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QPushButton" name="pushButton_settings">
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);</string>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4" colspan="3">
         <widget class="QPushButton" name="pushButton_parallelProjection">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);
</string>
          </property>
          <property name="text">
           <string>Parallel projection </string>
          </property>
         </widget>
        </item>
        <item row="4" column="4" colspan="3">
         <widget class="QPushButton" name="pushButton_centralProjection">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);
</string>
          </property>
          <property name="text">
           <string>Central projection</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="pushButton_easterEgg">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 150, 255);
font-size: 16px;
background-color: rgb(83, 95, 117);
</string>
          </property>
          <property name="text">
           <string>Information</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ViewModel</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">viewmodel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
